feat:add nuxt example (#6164)
prisma/prisma-examples: accelerate/nuxtjs-starter/README.md

### 3. Run a migration to create the `Quotes` table and seed the database

The Prisma schema file contains a single `Quotes` model. You can map this model to the database and create the corresponding `Quotes` table using the following command:

```
npx prisma migrate dev --name init
```

You now have an empty `Quotes` table in your database. Next, run the [seed script](./prisma/seed.js) to create some sample records in the table:

```
npx prisma db seed
```

### 4. Generate Prisma Client for Accelerate

When using Accelerate, Prisma Client doesn't need a query engine. That's why you should generate it as follows:

```
npx prisma generate --no-engine
```

### 5. Start the app

You can run the app with the following command:

```
npm run dev
```

Starting the development server will skip the prompt to install the Prisma CLI and create a
Prisma Schema as they already exist. It will also skip the prompt to  migrate the schema changes to the database, as the migrations folder already exists from step 3. You would then

> need to agree to the prompt to install and access Prisma Studio from the Nuxt Devtools.

Reviewer: ```suggestion
need to agree to the prompt to install and access Prisma Studio from the Nuxt Devtools:
```

```
Do you want to view and edit your data by installing Prisma Studio in Nuxt DevTools? › (Y/n)
  Select Y (yes) to install and use Prisma Studio in Nuxt Devtools.
```

### 6. Launch the app in the browser
Open the browser and navigate to http://localhost:3000

> You can see the performance and other stats (e.g. cache/hit) for the different Accelerate cache strategies at the bottom of the UI:

Reviewer: @ludralph please add a GIF here.

Reviewer 2: I have added it.

# Prisma Accelerate Example: Nuxt.js Starter

## Introduction

> The quote generator typically fetches and returns a random quote from a predefined set of quotes. In the context of your Nuxtjs application, the quote generator is a service that provides quotes, potentially with some caching strategy applied to optimize performance and reduce the load on the server.

Reviewer: ```suggestion
The quote generator retrieves the most recently added quote with and without caching enabled from a predefined set of quotes. In the context of your Nuxtjs application, the quote generator is a service that provides quotes, with some caching strategies applied to optimize performance and reduce the load on the database.
```

## General Steps of a Quote Generator:

> - Fetch a Random Quote: It selects a random quote from a collection of quotes.

Reviewer: As per the label the behaviour should be different.

The label in the landing page states: "Retrieves the most recently added quote with and without caching enabled"

![Uploading image.png…]()

Reviewer: ```suggestion
- Fetch a quote: Retrieves the most recently added quote with and without caching enabled.
```

> - Apply Caching Strategy: Depending on the request, it may apply different caching strategies (e.g., TTL, SWR, or no caching).

Reviewer: ```suggestion
- Apply caching strategy: Depending on the request, it may apply different caching strategies (e.g., TTL, SWR, TTL+SWR or no caching).
```

> - Return the Quote: The selected quote is returned, possibly along with metadata about the caching status and other relevant information.

Reviewer: ```suggestion
- Return the quote: The selected quote is returned, along with metadata about the caching status and other relevant information.
```

NUXT_PUBLIC_URL="http://localhost:3000"
```

Note that `__YOUR_DATABASE_CONNECTION_STRING__` and `__YOUR_ACCELERATE_CONNECTION_STRING__` are placeholder values that you need to replace with the values of your database and Accelerate connection strings. Notice that the Accelerate connection string has the following structure: `prisma://accelerate.prisma-data.net/?api_key=__YOUR_ACCELERATE_API_KEY__`.

### 3. Run a migration to create the `Quotes` table and seed the database

The Prisma schema file contains a single `Quotes` model. You can map this model to the database and create the corresponding `Quotes` table using the following command:

```
npx prisma migrate dev --name init
```

You now have an empty `Quotes` table in your database. Next, run the [seed script](./prisma/seed.js) to create some sample records in the table:

```
npx prisma db seed
```

### 4. Generate Prisma Client for Accelerate

When using Accelerate, Prisma Client doesn't need a query engine. That's why you should generate it as follows:

```
npx prisma generate --no-engine
```

### 5. Start the app

You can run the app with the following command:

```
npm run dev
```

> ### 6. ? Do you want to view and edit your data by installing Prisma Studio in Nuxt DevTools? › (Y/n)

Reviewer: Provide more context on this and put this as a sub-step under "Start the app". Similar to [here](https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/prisma-nuxt-module#getting-started).
